src/OrchardCore.Modules/OrchardCore.Workflows/Controllers/WorkflowTypeController.cs

```diff
@@ -128,7 +128,7 @@ public async Task<IActionResult> Index(WorkflowTypeIndexOptions options, PagerPa
             var sqlBuilder = dialect.CreateBuilder(_session.Store.Configuration.TablePrefix);
             sqlBuilder.Select();
             sqlBuilder.Distinct();
-            sqlBuilder.Selector(nameof(WorkflowIndex.WorkflowTypeId));
+            sqlBuilder.Selector(nameof(WorkflowIndex), nameof(WorkflowIndex.WorkflowTypeId), _session.Store.Configuration.Schema);
```

Comment:
Good catch! There are 4 other places in OC where with use `Selector` without providing a schema. `DbConnectionValidator` (2 times), `IndexingTaskManager`, and `SqlParser`. Can you please update these files also?